hw/bsp/nrf52dk/pkg.yml

```diff
@@ -39,8 +39,8 @@ pkg.cflags:
     - '-DPDM_ENABLED=0'
     - '-DPERIPHERAL_RESOURCE_SHARING_ENABLED=1'
     - '-DPWM0_ENABLED=1'
-    - '-DPWM1_ENABLED=0'
-    - '-DPWM2_ENABLED=0'
+    - '-DPWM1_ENABLED=1'
+    - '-DPWM2_ENABLED=1'
```

Comment:
I wonder if we could make these SDK flags conditional, such that PWMx_ENABLED would be set only if the matching os_dev for that PWM is enabled with that BSP.
I.e. in hw/bsp/nrf52dk/pkg.yml, you would have:
pkg.cflags.PWM0_ENABLED: -DPWM0_ENABLED
and in this file you'd have pkg.cflags: -DPWM0_ENABLED.

It *should* work, as newt orders the cflags passed to command line, although it might be a bit too hacky. I.e. depends on newt doing the cflag ordering. Others might have comments regarding this approach?